lib/src/data/chat/chat_joinning.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:misskey_dart/src/converters/date_time_converter.dart';
+import 'package:misskey_dart/src/data/base/user.dart';
+import 'package:misskey_dart/src/data/chat/chat_room.dart';
+
+part 'chat_joinning.freezed.dart';
+part 'chat_joinning.g.dart';
+
+@freezed
+class ChatJoining with _$ChatJoining {
+  const factory ChatJoining({
+    required String id,
+    @DateTimeConverter() required DateTime createdAt,
+    String? userId,
```

Comment:
```suggestion
    required String userId,
```

ref: https://github.com/misskey-dev/misskey/blob/2025.4.0-beta.0/packages/backend/src/models/json-schema/chat-room-invitation.ts#L18-L21
ref: https://github.com/misskey-dev/misskey/blob/2025.4.0-beta.0/packages/backend/src/models/json-schema/chat-room-membership.ts#L18-L21